src/js/pages/organizer/Calendar.jsx

```diff
@@ -1,29 +1,112 @@
 import React from "react";
 import {BootstrapTable, TableHeaderColumn} from 'react-bootstrap-table';
+import Select from 'react-select';
 import EventStore from "stores/EventStore";
 var moment = require('moment');
 
+
+class StreamEditor extends React.Component {
+  constructor(props) {
+    super(props);
+    this.updateData = this.updateData.bind(this);
+    console.log(props);
+    this.state = {
+      stream: props.defaultValue.stream,
```

Comment:
this feels odd. why the default value over the value? I dont see where the default value is defined.